src/inference/include/openvino/runtime/properties.hpp

```diff
@@ -642,9 +643,9 @@ struct NumStreams {
         NUMA = -2,  //!< Creates as many streams as needed to accommodate NUMA and avoid associated penalties
     };
```

Comment:
can we hide these values from end users? to prevent possible errors

Reply:
Yes but not in this PR. Only when we fix other plugins (like openvino_contrib) and demos